Clean up next steps screen

johnschoeman opened the pull request:
This commit is a minor clean up of some visual issues on the
'Next Steps' screen.

### Before
 
<img width="584" alt="Screen Shot 2020-06-19 at 12 55 09 PM" src="https://user-images.githubusercontent.com/16049495/85160605-90734500-b22c-11ea-84e7-243e65389168.png">

### After

<img width="584" alt="Screen Shot 2020-06-19 at 12 51 07 PM" src="https://user-images.githubusercontent.com/16049495/85160638-949f6280-b22c-11ea-8755-669c84285761.png">

JacobJaffe commented on app/views/ExposureHistory/NextSteps.tsx:
```diff
@@ -59,18 +57,14 @@ const NextSteps = (): JSX.Element => {
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```
Why the switch to percents here?

johnschoeman replied:
This was written before your pr that fixed the layout stuff with NavigationWrapper.  Flex is what we want now.

JacobJaffe commented on app/views/ExposureHistory/NextSteps.tsx:
```diff
@@ -82,6 +76,7 @@ const styles = StyleSheet.create({
   },
   button: {
     ...Buttons.largeBlue,
+    borderWidth: 1,
```
does this have an explicit color associated with the button spread? This border width seems like it could add an unwanted color as is?

johnschoeman replied:
yup! this should be removed.

JacobJaffe commented on ios/COVIDSafePaths.xcodeproj/project.pbxproj:
```diff
@@ -2130,9 +2130,9 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.pathcheck.covid-safepaths";
```
To check, were these changes meant to be bundled into this PR?

johnschoeman replied:
nope!

mattThousand commented on ios/COVIDSafePaths.xcodeproj/project.pbxproj:
```diff
@@ -2234,9 +2234,9 @@
-				PRODUCT_BUNDLE_IDENTIFIER = "org.pathcheck.covid-safepaths";
+				PRODUCT_BUNDLE_IDENTIFIER = org.pathcheck.bt;
 				PRODUCT_NAME = BT;
-				PROVISIONING_PROFILE_SPECIFIER = "match Development org.pathcheck.covid-safepaths";
```
Is this provisioning profile switch meant to be included?

johnschoeman replied:
nope!

johnschoeman committed:
Clean up next steps screen.

This commit is a minor clean up of some visual issues on the
'Next Steps' screen.

JacobJaffe reviewed (approved):
LGTM!
